colabs/wandb_registry/zoo_wandb.ipynb

```diff
@@ -0,0 +1,634 @@
+{
```

Comment:
<blockquote>From Noa's suggestion to use core registries instead:</blockquote><p>The goal of this notebook is to demonstrate how you and other members of your organization can use W&amp;B Registry to track, share, and use dataset and model artifacts in your machine learning workflows. By the end of this notebook, you will know how to use W&amp;B to:</p><ol><li>Use a&nbsp;<a href="https://docs.wandb.ai/guides/registry/registry_types#core-registry" target="_blank">core registry</a></li><li>Create&nbsp;<a href="https://docs.wandb.ai/guides/registry/create_collection" target="_blank">collections</a>&nbsp;within a registry</li><li>Make dataset and model artifacts available to other members of your organization, and</li><li>Download artifacts from the registry for inference</li></ol><p>To achieve this, we will train a neural network to identify animal classes, as an example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/560/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>